internal/cli/email_templates.go

```diff
@@ -192,12 +192,23 @@ func updateEmailTemplateCmd(cli *cli) *cobra.Command {
 			}
 
 			var oldTemplate *management.EmailTemplate
+			templateExists := true
 			err := ansi.Waiting(func() (err error) {
 				oldTemplate, err = cli.api.EmailTemplate.Read(apiEmailTemplateFor(inputs.Template))
 				return err
 			})
 			if err != nil {
-				return fmt.Errorf("failed to get the email template '%s': %w", inputs.Template, err)
+				if mErr, ok := err.(management.Error); ok && mErr.Status() != http.StatusNotFound {
```

Comment:
Just to check, is there any case where `err` is not a `management.Error`?

Reply:
Good callout, if we have network issues then the error won't be a `management.Error`. Fixed this in [`ed56973` (#611)](https://github.com/auth0/auth0-cli/pull/611/commits/ed569733b372e6c56196324d731562b99cc4c1d2)